### DISCUSSION TITLE
Add synchronization status and Force to sync button

### DISCUSSION THREAD

**Discussion by hiangeel** [glushchenko/fsnotes#1249] [Ideas]
Hi,
I have just come to FSNotes. I also downloaded its mobile app. 
I see that when a note was created on Mac, I have to close the mobile app and re-launch to got that note. Please add a small region that show synchronization status as well as a button to synchronize notes. Joplin is a good example for my suggestion.
One more thing, I can create many folders and sub-folders but they all are showed as the same level on mobile app. Please take a look at this.
Finally, do you have plan to support other cloud services like Dropbox, Onedrive or Google Drive?
Thank you.

**Comment by gingerbeardman** [comment]
> Finally, do you have plan to support other cloud services like Dropbox, Onedrive or Google Drive?

FSNotes macOS version supports any storage, but iOS version only supports iCloud https://github.com/glushchenko/fsnotes/issues/432

**Comment by glushchenko** [comment]
Sync started automatically on iOS and macOS, so no sense in this.
